### PR TITLE
Add scope - handle data in parent + initial tag search

### PR DESCRIPTION
**Pull request by kfaln**

### CODE REVIEW

File: src/modules/Preservation/http/PreservationApiClient.ts [hunk 1]
```diff
@@ -118,8 +124,29 @@ class PreservationApiClient extends ApiClient {
             id: 2
         }]);
     }
-}
 
+    async getTagsForAddPreservationScope(tagNo: string, setRequestCanceller?: RequestCanceler): Promise<TagData[]> {
+        // const endpoint = '/Tags';
```
Comment (Cnordbo) [inline src/modules/Preservation/http/PreservationApiClient.ts]: `/Tags/Search`

We also need to setup support for sending in Project that should be set as query string.

File: src/modules/Preservation/views/AddScope/types.ts [hunk 2]
```diff
@@ -1,3 +1,9 @@
 export type Tag = {
     id: number;
 }
+
+export type TagRow = {
+    tagId: number;
+    tagNo: string;
+    description: string;
+}
```
Comment (Cnordbo) [inline src/modules/Preservation/views/AddScope/types.ts]: Only needed by Table in SelectTags, can be removed from common types.

File: src/modules/Preservation/views/AddScope/SelectTags.tsx [hunk 3]
```diff
@@ -1,69 +1,46 @@
...
 type SelectTagsProps = {
-    tags: Tag[];
+    selectedTags: Tag[];
+    scopeTableData: TagRow[];
     setSelectedTags: (tags: Array<Tag>) => void;
+    searchTags: (tagNo: string | null) => void;
     nextStep: () => void;
 }
 
-type TagRow = {
```
[the remark this answers is not in the document]
Comment (kfaln) [reply src/modules/Preservation/views/AddScope/SelectTags.tsx]: It's actually used in both AddScope (where the data resides in state) and SelectTags (where the data is presented in Table). I'm gonna hold on to it a while longer in common types.

File: src/modules/Preservation/http/PreservationApiClient.ts (hunk 1 above)
Comment (kfaln) [reply src/modules/Preservation/http/PreservationApiClient.ts]: Yes, will look into this next.

**Pull request merged**